### PR TITLE
paint_board-2: Add enum for main colors

### PR DESCRIPTION
**Pull request by KoTuK2306**

### CODE REVIEW

File: src/enums/colors-enum.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  maroon = "#800000",
+  navy = "#000080",
+  olive = "#808000",
+  purple = "#800080",
+  red = "#FF0000",
+  silver = "#C0C0C0",
+  tan = "#D2B48C",
+  white = "FFFFFF",
+  yellow = "#FFFF00",
+}
```
Comment (pavelsokolov21) [inline src/enums/colors-enum.ts]: добавь пустую строку в конце файла

File: src/enums/colors-enum.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+export enum colors {
```
Comment (pavelsokolov21) [inline src/enums/colors-enum.ts]: emun'ы пишутся с большой буквы

**Review by pavelsokolov21 — CHANGES_REQUESTED** [review]

File: src/enums/colors-enum.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  aqua = "#00FFFF",
+  black = "000000",
```
Comment (pavelsokolov21) [inline src/enums/colors-enum.ts]: нет #

File: src/enums/colors-enum.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  green = "#008000",
+  lime = "#00FF00",
+  maroon = "#800000",
+  navy = "#000080",
+  olive = "#808000",
+  purple = "#800080",
+  red = "#FF0000",
+  silver = "#C0C0C0",
+  tan = "#D2B48C",
+  white = "FFFFFF",
```
Comment (pavelsokolov21) [inline src/enums/colors-enum.ts]: нет #

**Review by pavelsokolov21 — CHANGES_REQUESTED** [review]

File: src/enums/colors-enum.ts (hunk 1 above)
Comment (IliaSolovev) [reply src/enums/colors-enum.ts]: I think you should add a rule in your redactor that in the future it was automatic

Comment (IliaSolovev) [reply src/enums/colors-enum.ts]: also, add config rule that eslint reformat code after save

**Review by IliaSolovev — CHANGES_REQUESTED** [review]

File: src/enums/colors-enum.ts (hunk 1 above)
Comment (pavelsokolov21) [reply src/enums/colors-enum.ts]: I'll do it in pre-commit

**Comment by KoTuK2306** [comment]
@IliaSolovev @SMPlay Check

**Review by pavelsokolov21 — APPROVED** [review]

**Review by IliaSolovev — APPROVED** [review]

**Pull request merged**